litmus-portal/graphql-server/pkg/graphql/mutations/mutation.go

```diff
@@ -33,15 +41,10 @@ func ClusterRegister(input model.ClusterInput) (*model.ClusterRegResponse, error
 		PlatformName: input.PlatformName,
 		CreatedAt:    strconv.FormatInt(time.Now().Unix(), 10),
 		UpdatedAt:    strconv.FormatInt(time.Now().Unix(), 10),
+		Token:        token,
 	}
 
-	err := database.InsertCluster(newCluster)
-	if err != nil {
-		return &model.ClusterRegResponse{}, err
-	}
-
-	log.Print("NEW CLUSTER REGISTERED : ID-", newCluster.ClusterID, " PID-", newCluster.ProjectID)
```

Comment:
We should keep this for debugging purpose

Reply:
It is there just change the flow

Reply:
@gdsoumya We need it for showing the details of the cluster on target screens.